dependencies.yaml

```
@@ -115,9 +117,50 @@ dependencies:
         packages:
           - numpy>=1.23,<2.0a0
           - pynvml>=11.4.1
+  depends_on_ucx_build:
+    common:
+      - output_types: conda
+        packages:
+          - &ucx_conda_build ucx==1.15.0
```

It doesn't look like this reference is ever used anywhere.
```suggestion
          - ucx==1.15.0
```

---

blegh yeah this is left over from an earlier version. Thank you!